src/components/Modals/AddClientModal.jsx

```diff
@@ -117,6 +119,7 @@ const AddClientModal = ({ showAddClientModal, setShowAddClientModal }) => {
         state,
         dispatch
       );
+      notify.addNotification('success', `this is a test`);
```

Comment:
Rather than using the statement: `this is a test`, we could use the notification message from the notification state from the hook.